packages/lib-classifier/src/plugins/tasks/DrawingTask/models/DrawingAnnotation.spec.js

```diff
@@ -1,19 +1,187 @@
+import { types } from 'mobx-state-tree'
+import DrawingTask from './DrawingTask'
 import DrawingAnnotation from './DrawingAnnotation'
 import { Point } from '@plugins/drawingTools/models/marks'
 
-const point = Point.create({ id: 'mockAnnotation', frame: 0, toolIndex: 0, toolType: 'point', x: 100, y: 150 })
+describe('Model > DrawingAnnotation', function () {
+  const point = Point.create({ id: 'mockAnnotation', frame: 0, toolIndex: 0, toolType: 'point', x: 100, y: 150 })
 
-const drawingAnnotationSnapshot = {
-  id: 'drawing1',
-  task: 'T0',
-  taskType: 'drawing',
-  value: [ point.id ]
-}
 
-describe('Model > DrawingAnnotation', function () {
+  const drawingAnnotationSnapshot = {
+    id: 'drawing1',
+    task: 'T0',
+    taskType: 'drawing',
+    value: [ point.id ]
+  }
+
   it('should exist', function () {
     const drawingAnnotation = DrawingAnnotation.create(drawingAnnotationSnapshot)
     expect(drawingAnnotation).to.exist()
     expect(drawingAnnotation).to.be.an('object')
   })
+
+  describe('annotation snapshots', function () {
+    describe('without subtasks', function () {
+      let snapshot
+
+      before(function () {
+        const drawingTask = DrawingTask.create({
+          instruction: 'draw things!',
+          taskKey: 'T0',
+          tools: [
+            {
+              type: 'point'
+            }
+          ],
+          type: 'drawing'
+        })
+        const pointTool = drawingTask.tools[0]
+        const point = pointTool.createMark({
+          x: 50,
+          y: 100
+        })
+        const annotation = DrawingAnnotation.create({
+          id: 'drawing1',
+          task: 'T0',
+          taskType: 'drawing',
+          value: [ point ]
+        })
+        const mockStore = types.model('MockStore', {
+          drawingTask: DrawingTask,
+          annotation: DrawingAnnotation
+        })
+        .create({
+          drawingTask,
+          annotation
+        })
+        snapshot = annotation.toSnapshot()
+      })
+
+      it('should contain exactly one annotation', function () {
+        expect(snapshot).to.have.lengthOf(1)
```

Comment:
I'd like the expectation here to check the type since snapshots are doing type conversions. Strings can also have length. I think with chai, these can be chained, so `expect(snapshot).to.be.an('array').that.has.lengthOf(1)`

Reply:
Drawing annotations don't do any type conversion. The snapshot is a flattened array of annotations, per ADR 25. I think we're getting confused here with classification snapshots, or maybe mark snapshots?

Reply:
The set of tests after this do test the snapshot in detail, and will fail if it sin't an array.